quoting-service/values.yaml

```diff
@@ -573,9 +573,9 @@ quoting-service-handler:
       enabled: false
       type: redis
       proxyConfig:
-        host: redis
```

Comment:
Don't you want to bump quoting service version like ALS?

Reply:
but it's already set to a proper value:
![image](https://github.com/user-attachments/assets/a7e2628b-b335-42c8-b735-c3ec5f948311)
![image](https://github.com/user-attachments/assets/4155b45d-ef3e-44a5-bc60-9547e8b67f70)

Reply:
Ok